domenetjenester/dokumentbestiller/src/main/java/no/nav/foreldrepenger/dokumentbestiller/infobrev/SendInformasjonsbrevPåminnelseTask.java

```diff
@@ -72,20 +84,31 @@ public void doTask(ProsessTaskData prosessTaskData) {
             return; // Unngå brev til død bruker
         }
 
-        var fagsak = fagsakRepository.finnEksaktFagsakReadOnly(Long.parseLong(prosessTaskData.getPropertyValue(FAGSAK_ID_KEY)));
-        var brukEnhet = behandlendeEnhetTjeneste.finnBehandlendeEnhetFor(fagsak);
+        var fagsakFar = fagsakRepository.finnEksaktFagsakReadOnly(Long.parseLong(prosessTaskData.getPropertyValue(FAGSAK_ID_KEY)));
+
+        //Sjekk at det finnes fedrekvote igjen
+        var fagsakRelasjon = fagsakRelasjonRepository.finnRelasjonFor(fagsakFar);
+        var fagsakMor = fagsakRelasjon.getRelatertFagsak(fagsakFar).orElseThrow();
+        var behandlingMor = behandlingRepository.finnSisteAvsluttedeIkkeHenlagteBehandling(fagsakMor.getId()).orElseThrow();
```

Comment:
Vi har en egen tjeneste for å hente annen parts gjeldendebehandling RelatertBehandlingTjeneste.hentAnnenPartsGjeldendeVedtattBehandling.
Kanskje enklere å bruke den

Reply:
Enig. Har oppdatert